src/model/AirConditionerVariableRefrigerantFlow.cpp

```diff
@@ -91,7 +91,7 @@ namespace detail {
 
   const std::vector<std::string>& AirConditionerVariableRefrigerantFlow_Impl::outputVariableNames() const
   {
-    static std::vector<std::string> result{ "VRF Heat Pump Total Cooling Rate",
+    static const std::vector<std::string> result{ "VRF Heat Pump Total Cooling Rate",
```

Comment:
A ton of changes (most of them probably) are similar to this one and deal with all children of ModelObject with regards to `outputVariableNames()`